ci/jenkins/jobs/macros.yaml

```diff
@@ -7,6 +7,13 @@
           jenkins-jobs update -r ci/jenkins/jobs
           rm ci/jenkins/jobs/defaults.yaml
 
+- builder:
+    name: builder-rdiff-backup
```

Comment:
maybe also renamed this builder-rdiff-backup to builder-jenkins-backup

Reply:
Renamed to builder-jenkins-backup, thanks.